Lib/test/test_flufl.py

```diff
@@ -16,21 +15,21 @@ def test_barry_as_bdfl(self):
                     __future__.CO_FUTURE_BARRY_AS_BDFL)
         self.assertRegex(str(cm.exception),
                          "with Barry as BDFL, use '<>' instead of '!='")
-        self.assertEqual(cm.exception.text, '2 != 3\n')
+        self.assertEqual(cm.exception.text, '2 != 3')
         self.assertEqual(cm.exception.filename, '<FLUFL test>')
         self.assertEqual(cm.exception.lineno, 2)
-        self.assertEqual(cm.exception.offset, 4)
+        self.assertEqual(cm.exception.offset, 3)
```

Comment:
Because of this change (and the corresponding one below) the test will now fail with `-X oldparser`. I'm not sure if we are trying to keep the tests passing with both parsers? (FWIW I think the new parser does a better job here. :-)

Reply:
> Because of this change (and the corresponding one below) the test will now fail with `-X oldparser`.

Good point, I suppose that I will need to give a range for the offset instead and add a comment explaining that the range is the token (start/end).

> (FWIW I think the new parser does a better job here. :-)

Agreed :)